libgnucash/engine/Account.cpp

```diff
@@ -2168,17 +2146,17 @@ xaccAccountInsertLot (Account *acc, GNCLot *lot)
 /********************************************************************\
 \********************************************************************/
 static void
-xaccPreSplitMove (Split *split, gpointer dummy)
+xaccPreSplitMove (Split *split)
 {
     xaccTransBeginEdit (xaccSplitGetParent (split));
 }
 
 static void
-xaccPostSplitMove (Split *split, Account *accto)
+xaccPostSplitMove (Split *split)
 {
     Transaction *trans;
 
-    xaccSplitSetAccount(split, accto);
+    xaccSplitSetAccount(split, nullptr);
```

Comment:
> 1, that's a [no-op](https://github.com/Gnucash/gnucash/blob/c1ee59c39bf5287f30dc2ef489b5f4b71db66029/libgnucash/engine/Split.cpp#L978) and 2, the whole point of `xaccMoveSplits` is to change the splits' account to accto. You've broken it.

it's because `xaccPostSplitMove` was being called with `accto==nullptr` in the only caller below.

Reply:
> it's because `xaccPostSplitMove` was being called with `accto==nullptr` in the only caller below.

uh... fixednow.